src/components/pool/DepositTab.tsx

```diff
@@ -407,19 +437,19 @@ export default function DepositTab(props: DepositTabProps) {
         onCancel={() => {
           setIsTransactionPending(false);
         }}
-        estimatedTotal='$5,038'
+        estimatedTotal={usdEstimate}
         token0Ticker={drawData.token0Label}
         token1Ticker={drawData.token1Label}
         token0Estimate={token0Amount}
         token1Estimate={token1Amount}
-        numberOfShares='2'
+        numberOfShares={sharesEstimate}
         maxSlippage={maxSlippage}
-        networkFee='0.01'
+        networkFee='TODO'
```

Comment:
Are we leaving this as a TODO?

Reply:
Yes, it is not actually rendered in the component.